target/linux/ath79/dts/qca9563_tplink_tl-wpa8630p-v2.dtsi

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "tp-link:green:power";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi2g {
+			label = "tp-link:green:wifi2g";
+			gpios = <&gpio 19 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		wifi5g {
+			label = "tp-link:green:wifi5g";
+			gpios = <&gpio 21 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		lan {
+			label = "tp-link:green:lan";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+
+		led_control {
+			gpio-export,name = "tp-link:led:control";
+			gpios = <&gpio 14 GPIO_ACTIVE_HIGH>;
```

Comment:
What happened to `gpio-export,output` ?

Reply:
Good question - As I don't currently have access to the device, I cannot resolve this for the time being (and I don't remember what it should be set to by default).

Reply:
Also, what's the purpose of this? I don't quite get why a GPIO export should be used to control a LED, especially that there is a node for that already.

Reply:
I just tested this part (sorry about the delay!). Currently this is the case:
```
# echo 1 > /sys/class/gpio/tp-link\:led\:control/value 
ash: write error: Operation not permitted
# cat /sys/kernel/debug/gpio 
...
 gpio-14  (                    |tp-link:led:control ) in  lo 
```

----------------

This patch fixes the output, 
```patch 
 		led_control {
 			gpio-export,name = "tp-link:led:control";
-			gpios = <&gpio 14 GPIO_ACTIVE_HIGH>;
+			gpio-export,output = <0>;
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
 		};
 	};
 };
```

After booting, this is the result. The LEDs are on by default after boot:
```
# cat /sys/kernel/debug/gpio
gpio-14  (                    |tp-link:led:control ) out lo ACTIVE LOW
$ cat /sys/class/gpio/tp-link\:led\:control/value 
1
```

Changing the LEDs
```
# Turning off LEDs
echo 0 > /sys/class/gpio/tp-link\:led\:control/value
# Turning on LEDs
echo 1 > /sys/class/gpio/tp-link\:led\:control/value
```

Reply:
Thanks for finding this issue again. Please send a patch for it (though I'm never sure which combination of ACTIVE_X and output = y is the correct one....)